### PR TITLE
Add JSON Tile3D examples to Playground

### PR DESCRIPTION
**Pull request by ibgreen**
@xintongxia @jianhuang01 These examples can be tested in `examples/playground`. 

They don't quite work, some token issue and I think the Draco loader is not getting used.

I think we can fix them, it would be quite neat to have these examples as they are very minimal and clean.

For #

#### Change List
-

depends on #3606

### CODE REVIEW

File: examples/playground/json-examples/3d-tiles-cairo-vricon.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "type": "MapView",
+      "mapStyle": "mapbox://styles/mapbox/dark-v9",
+      "controller": true
+    }
+  ],
+  "layers": [
+    {
+      "type": "Tile3DLayer",
+      "id": "tiles-cairo-vricon",
+      "ionAssetId": 29328,
```
Comment (Pessimistress) [inline examples/playground/json-examples/3d-tiles-cairo-vricon.json]: I thought we made the ion props experimental?

File: docs/layers/tile-3d-layer.md [hunk 2]
```diff
@@ -118,6 +123,16 @@ Along with other options as below,
...
 
+## Sub Layers
+
+The Tile3DLayer renders the following sublayers based on tile [format](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#introduction):
+
+* `scenegraph` - a [ScenegraphLayer](/docs/layers/scenegraph-layer.md) rendering all the tiles with Batched 3D Model format or Instanced 3D Model format.
+  - `_lighting` is default to `pbr`.
+* `pointcloud` - a [PointCloudLayer](/docs/layers/point-cloud-layer.md) rendering all the tiles with Point Cloud format.
+  - `pointSize` is default to `1.0`.
+  - `getColor` is default to `[255, 0, 0]`
```
Comment (ibgreen) [inline docs/layers/tile-3d-layer.md]: This is an unusual default color. It is only used if no color in the cloud right, maybe we need to state that?

File: docs/upgrade-guide.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 - Previously deprecated `Layer` class method, `screenToDevicePixels`, is removed. Use luma.gl [utility methods](https://luma.gl/#/documentation/api-reference/webgl-2-classes/device-pixels) instead.
 
+- `ScreenGridLayer` is now limited to browsers that support WebGL2 or `OES_texture_float` extension.
```
Comment (ibgreen) [inline docs/upgrade-guide.md]: Move to different PR?

File: examples/playground/src/app.js [hunk 4]
```diff
@@ -66,6 +62,23 @@ export class App extends Component {
     const jsonProps = this.jsonConverter.convert(json);
     this._updateViewState(jsonProps);
 
+    jsonProps.layers = jsonProps.layers.map(layer => {
+      // TODO @deck.gl/json module support replacing non-instantiated class
+      const dracoProps = {};
```
Comment (ibgreen) [inline examples/playground/src/app.js]: Replace with `configuration.constants` as discussed

File: examples/playground/src/configuration.js [hunk 5]
```diff
@@ -9,6 +9,12 @@ import * as MeshLayers from '@deck.gl/mesh-layers';
 import {COORDINATE_SYSTEM} from '@deck.gl/core';
 import GL from '@luma.gl/constants';
 
+import {registerLoaders} from '@loaders.gl/core';
```
Comment (ibgreen) [inline examples/playground/src/configuration.js]: Why is this here? need to rebase to latest master?

Comment (xintongxia) [reply examples/playground/src/configuration.js]: I did global search and couldn't find any places `CSVLoader` is registered.

File: docs/layers/tile-3d-layer.md [hunk 6]
```diff
@@ -131,7 +131,7 @@ The Tile3DLayer renders the following sublayers based on tile [format](https://g
   - `_lighting` is default to `pbr`.
 * `pointcloud` - a [PointCloudLayer](/docs/layers/point-cloud-layer.md) rendering all the tiles with Point Cloud format.
   - `pointSize` is default to `1.0`.
-  - `getColor` is default to `[255, 0, 0]`
+  - `getColor` is default to `[255, 0, 0]`. This value is only applied when no [color properties](https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/specification/TileFormats/PointCloud/README.md#point-colors) are defined in point cloud tile file.
```
Comment (Pessimistress) [inline docs/layers/tile-3d-layer.md]: Move this to the PR that addresses API audit comments

File: test/modules/json/data/deck-props.json [hunk 7]
```diff
@@ -21,6 +21,7 @@
   "layers": [
     {
       "type": "ScatterplotLayer",
+      "id": "SCATTER_PLOT_LAYER_ID",
```
Comment (Pessimistress) [inline test/modules/json/data/deck-props.json]: ?? Use something meaningful

File: examples/playground/json-examples/3d-tiles-royal.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
+{
+  "description": "The deck.gl website tile-3d-layer example",
+  "websiteUrl": "https://deck.gl/#/examples/geo-layers/tile-3d-layer",
+  "initialViewState": {
+    "latitude": -37.80581839466932,
+    "longitude": 144.97127808879253,
```
Comment (Pessimistress) [inline examples/playground/json-examples/3d-tiles-royal.json]: What's with these numbers, do we need these many decimals?

File: examples/playground/json-examples/3d-tiles-new-york.json [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+      "type": "MapView",
+      "mapStyle": "mapbox://styles/mapbox/dark-v9",
+      "controller": true
+    }
+  ],
+  "layers": [
+    {
+      "type": "Tile3DLayer",
+      "id": "tiles-new-york",
+      "DracoLoader": "DracoLoader",
```
Comment (xintongxia) [inline examples/playground/json-examples/3d-tiles-new-york.json]: gltf parser doesn't work with `DracoWorkerLoader` @ibgreen

Comment (ibgreen) [reply examples/playground/json-examples/3d-tiles-new-york.json]: Yes this needs to be fixed in loaders.gl. The new GLTFParser being developed for loaders.gl v2 should handle it but still has some bug.

File: examples/playground/src/configuration.js (hunk 5 above)
Comment (ibgreen) [reply examples/playground/src/configuration.js]: OK you moved it from app.js to here, that is good.
